content/chat/getting-started/react.textile

```diff
@@ -0,0 +1,842 @@
+          {roomReactions.map((r, idx) => (
+            <span
+              key={idx}
+              className="px-2 py-1 bg-white rounded text-blue-600"
+            >
+              {r.type}
+            </span>
+          ))}
+        </div>
+      </div>
+    </div>
+  );
+}
+```
+
+* Finally, add the @ReactionComponent@ component to your main app component:
+
+```[react]
+// App.tsx
+function App() {
+  // Wrap your Room component with the ChatRoomProvider:
+  return (
+    <ChatRoomProvider
+      id="my-first-room" // The room ID you want to create or join
+      release={true} // Release the room automatically when unmounted
+      attach={true} // Attach to the room automatically when mounted
+      options={AllFeaturesEnabled}
+    >
+      <div className="flex flex-col w-[900px] h-full border-1 border-blue-500 rounded-lg overflow-hidden mx-auto font-sans">
+        <div className="flex flex-row w-full border-1 border-blue-500 rounded-lg overflow-hidden mx-auto font-sans">
+          <div className="flex-1 border-1 border-blue-500">
+            <ConnectionStatus />
+          </div>
+          <div className="flex-1 border-1 border-blue-500">
+            <RoomStatus />
+          </div>
+        </div>
+        <div className="flex flex-1 flex-row justify-evenly">
+          <div className="flex flex-col w-1/2 border-1 border-blue-500 rounded-lg overflow-hidden mx-auto font-sans">
+            <div className="flex-1 border-1 border-blue-500 overflow-y-auto">
+              <PresenceStatus />
+            </div>
+            <div className="flex-1 border-1 min-h-[100px] max-h-[100px] border-blue-500">
+            {/* Your ReactionComponent component should go here */}
+              <ReactionComponent />
+            </div>
+          </div>
+          <div className="flex flex-col bg-white w-1/2 border-1 border-blue-500 rounded-lg overflow-hidden mx-auto font-sans">
+            <ChatBox />
+          </div>
+        </div>
+      </div>
+    </ChatRoomProvider>
+  );
+}
+```
+
+The above code should display a list of reactions that can be sent to the room. When you click on a reaction, it will send it to the room and display it in the UI.
+
+* You can also send a reaction to the room via the Ably CLI by running the following command:
+
+```[sh]
+  ably rooms reactions send my-first-room 👍
+```
+
+h2(#next). Next steps
+
+Continue exploring Ably Chat with React:
+
+Read more about the concepts covered in this guide:
+* Read more about using "rooms":/docs/chat/rooms and sending "messages":/docs/chat/rooms/messages.
+* Find out more regarding "online status":/docs/chat/rooms/presence.
+* Understand how to use "typing indicators":/docs/chat/rooms/typing.
+* Send "reactions":/docs/chat/rooms/reactions to your rooms.
+* Read into pulling messages from "history":/docs/chat/rooms/history and providing context to new joiners.
+* Understand "token authentication":/docs/authentication/token-authentication before going to production.
```

Comment:
We should add `?lang=react` to these links so that the pages open with React already selected.